build/toolchain/linux/BUILD.gn

```diff
@@ -17,29 +21,90 @@ if (use_goma) {
   compiler_prefix = ""
 }
 
+if (host_cpu == "arm64") {
+  rebased_clang_dir =
+      rebase_path("//buildtools/linux-arm64/clang/bin", root_build_dir)
+} else {
```

Comment:
Is the issue that `host_cpu` isn't reliably set to x64 on x64 machines?

IIRC that may have been the case since I think I tried to do so and it caused an issue that I then didn't dig into to resolve :)

Reply:
Mostly that it was the default before so I'm leaving it as the default after. It possible to setup something like Rosetta on Linux using QEMU.